closed/OpenJ9.gmk

```diff
@@ -485,6 +492,22 @@ run-preprocessors-j9 : $(OUTPUT_ROOT)/vm/cmake.stamp
 
 else # OPENJ9_ENABLE_CMAKE
 
+ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+  CUSTOM_COMPILER_ENV_VARS += JITSERVER_SUPPORT=1
+
+  ifneq (,$(OPENSSL_CFLAGS))
+    CUSTOM_COMPILER_ENV_VARS += OPENSSL_CFLAGS="$(OPENSSL_CFLAGS)"
+  endif
+
+  ifneq (,$(OPENSSL_DIR))
+    CUSTOM_COMPILER_ENV_VARS += OPENSSL_DIR="$(OPENSSL_DIR)"
+  endif
+
+  ifneq (,$(OPENSSL_BUNDLE_LIB_PATH))
+    CUSTOM_COMPILER_ENV_VARS += OPENSSL_BUNDLE_LIB_PATH="$(OPENSSL_BUNDLE_LIB_PATH)"
+  endif
+endif
```

Comment:
Please add `# OPENJ9_ENABLE_JITSERVER` to the end of this line to make it easier to recognize the matching `ifeq`.